build.go

```diff
@@ -37,37 +39,47 @@ func Build(executable Executable, credentialManager CredentialManager, logger sc
 		layer.Launch = true
 		layer.Build = true
 
-		buffer := bytes.NewBuffer(nil)
-		args := []string{"rev-parse", "HEAD"}
-		err = executable.Execute(pexec.Execution{
-			Args:   args,
-			Dir:    context.WorkingDir,
-			Stdout: buffer,
-			Stderr: buffer,
-		})
+		exist, err := fs.Exists(filepath.Join(context.WorkingDir, ".git"))
 		if err != nil {
-			logger.Detail(buffer.String())
-			return packit.BuildResult{}, fmt.Errorf("failed to execute 'git rev-parse HEAD': %w", err)
+			panic(err)
```

Comment:
We probably don't want to `panic` here.

Reply:
Woof! You are right about that.

Reply:
Alright that should be all squared away.